### PR TITLE
Scanner Package Updated Readme and Change Log

### PR DESCRIPTION
**Pull request by Ramkrishna-egov**

### CODE REVIEW

File: packages/referral_reconciliation/lib/blocs/referral_reconciliation_listeners.dart
Comment (naveenr-egov) [inline packages/referral_reconciliation/lib/blocs/referral_reconciliation_listeners.dart:1]: delete this file since it's not required

**Review by naveenr-egov — CHANGES_REQUESTED** [review]
resolve comments

**Review by naveenr-egov — DISMISSED** [review]
Reviewed

File: packages/inventory_management/test/blocs/inventory_report_test.dart
Comment (naveenr-egov) [inline packages/inventory_management/test/blocs/inventory_report_test.dart:1]: Test data has been moved to constant file

**Review by naveen-egov — APPROVED** [review]

**Pull request merged**